cipher-null/README.md

```diff
@@ -0,0 +1,5 @@
+# cipher-null
+
+---
+
+Provides a mechanism to encrypt and decrypt a string in Java.
```

Comment:
Is this definitely correct?  The use of the word null suggests to me that this implementation doesn't actually do encryption.